lib/graphql_devise.rb

```diff
@@ -20,22 +20,36 @@ def self.load_schema
     @schema_loaded = true
   end
 
-  def self.resource_mounted?(mapping_name)
-    @mounted_resources.include?(mapping_name)
+  def self.resource_mounted?(model)
+    @mounted_resources.include?(model)
   end
 
-  def self.mount_resource(mapping_name)
-    @mounted_resources << mapping_name
+  def self.mount_resource(model)
+    @mounted_resources << model
```

Comment:
we do on the resource_loader

Reply:
I think it would less error prone if the check is closer to the assignation, like with `add_mapping` bellow where the key is looked among the devise mapping before adding

Reply:
With `add_mapping` it is important to prevent adding a new Devise mapping as it has mnay implications, with this one, no real problem if we ever mount the same model more than once. I think it would just add unnecessary overhead to check the list each time we mount a resource. This array is only used to later check the model is or not already on the list.